.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is a Visual Studio Code specific file, and should probably be kept out of the repository.